src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -26,25 +26,29 @@ Connect Sentry to your AWS account to automatically add Sentry error and perform
 
    ![](project.png)
 
-1. Click on **Add Sentry's CloudFormation to your AWS**.
+1. Click on **Go To AWS**.
```

Comment:
So we can just tell people to Click, no need to Click on (fewer words increases readability)